sys/suit/coap.c

```diff
@@ -77,7 +77,10 @@ static uint8_t _manifest_buf[SUIT_MANIFEST_BUFSIZE];
 #ifdef MODULE_SUIT_V4
 static inline void _print_download_progress(size_t offset, size_t len, uint32_t image_size)
 {
-DEBUG("_suit_flashwrite(): writing %u bytes at pos %u\n", len, offset);
+    (void) offset;
+    (void) len;
+    (void) image_size;
+    DEBUG("_suit_flashwrite(): writing %u bytes at pos %u\n", len, offset);
```

Comment:
You also remove the `&& defined(MODULE_SUIT_V4)` since the function is already in a `MODULE_SUIT_V4` only block.

Reply:
unrelated but ok